universe/Effect.cpp

```diff
@@ -365,15 +366,14 @@ void SetMeter::Execute(const TargetsCauses& targets_causes, AccountingMap* accou
         const TargetsAndCause&              targets_and_cause     = targets_entry.second;
         TargetSet                           targets               = targets_and_cause.target_set;
 
-        if (log_verbose) {
-            DebugLogger() << "\n\nExecute SetMeter effect: \n" << Dump();
-            DebugLogger() << "SetMeter execute targets before: ";
-            for (std::shared_ptr<UniverseObject> target : targets)
-                DebugLogger() << " ... " << target->Dump();
-        }
 
-        if (!accounting_map && !log_verbose) {
```

Comment:
Changing `if (!accounting_map && !log_verbose)` to just `if (!accounting_map)` means that the following `else if (!accounting_map)` case will never execute. Something doesn't make sense here...